frontend/components/sections/BannersSection/SingleBanner.tsx

```diff
@@ -13,7 +13,7 @@ interface SingleBannerProps {
 
 export function SingleBanner({ id, banner }: SingleBannerProps) {
    return (
-      <Box as="section" id={id} position="relative" w="full" pt="36" pb="16">
+      <Box as="section" id={id} position="relative" w="full" py="144px">
```

Comment:
```suggestion
      <Box as="section" id={id} position="relative" w="full" minHeight="310px" display="flex" placeItems="center">
```
We can do this to make it dynamic. The minHeight is 1px less than the old padding height total (design OCD doesn't allow single-digit values 😅), and `display="flex" placeItems="center"` handle the centering.

We also have an opportunity to clean the code up elsewhere by replacing:
```
top="0"
left="0"
bottom="0"
right="0"
```
with the awesome new shorthand `inset`:
```
inset="0"
```